### ISSUE TITLE
Traceback (most recent call last) line 57,50,11,8,and 6.

### ISSUE THREAD

**Issue by TeSoad90** [jdholtz/auto-southwest-check-in#238]
### Version

Auto-Southwest Check-In v7.3

### Browser Version

Brave 1.63.169

### Description

When I run python3 southwest.py Conf# First Last I get the following. My flight is not until the 16th (5 days as of 3/11/24)

Traceback (most recent call last):
  File "C:\Windows\system32\auto-southwest-check-in\southwest.py", line 57, in <module>
    init(arguments)
  File "C:\Windows\system32\auto-southwest-check-in\southwest.py", line 50, in init
    from lib.main import main
  File "C:\Windows\system32\auto-southwest-check-in\lib\main.py", line 11, in <module>
    from .config import GlobalConfig, ReservationConfig
  File "C:\Windows\system32\auto-southwest-check-in\lib\config.py", line 8, in <module>
    from .utils import NotificationLevel, is_truthy
  File "C:\Windows\system32\auto-southwest-check-in\lib\utils.py", line 6, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'

![image](https://github.com/jdholtz/auto-southwest-check-in/assets/68580386/d6537ee8-763a-4c50-b8db-3f2290192264)
![Checkinscript](https://github.com/jdholtz/auto-southwest-check-in/assets/68580386/73670f2c-4b83-4ba6-b2ad-ecb55d1679ce)

### To Reproduce

python3 southwest.py Conf# First Last

### Expected Behavior

for it to be ready for check-in.

### Relevant logs and program output

```sh
Traceback (most recent call last):
  File "C:\Windows\system32\auto-southwest-check-in\southwest.py", line 57, in <module>
    init(arguments)
  File "C:\Windows\system32\auto-southwest-check-in\southwest.py", line 50, in init
    from lib.main import main
  File "C:\Windows\system32\auto-southwest-check-in\lib\main.py", line 11, in <module>
    from .config import GlobalConfig, ReservationConfig
  File "C:\Windows\system32\auto-southwest-check-in\lib\config.py", line 8, in <module>
    from .utils import NotificationLevel, is_truthy
  File "C:\Windows\system32\auto-southwest-check-in\lib\utils.py", line 6, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
```

### Additional context

I used this script last year around this time and it worked fine, I can't remember if it only worked if it was within X amount of days. I did run the upgrade script and downloaded the new python as I had 3.11

**Comment by TeSoad90** [comment]
when I pull logs the logs are all from last year April 18th.

2023-04-18 00:24:07 DEBUG MainProcess[log]: Initialized the application
2023-04-18 00:24:07 DEBUG MainProcess[main]: Called with 3 arguments
2023-04-18 00:24:08 DEBUG MainProcess[config]: Reading the configuration file
2023-04-18 00:24:08 DEBUG MainProcess[config]: No configuration file found. Using defaults
2023-04-18 00:24:08 DEBUG MainProcess[main]: Flight added through CLI arguments
2023-04-18 00:24:08 DEBUG MainProcess[main]: Monitoring 0 accounts and 1 flights
2023-04-18 00:24:09 DEBUG Process-1[flight_retriever]: Scheduling reservations for 1 flights
2023-04-18 00:24:09 DEBUG Process-1[checkin_scheduler]: No headers set. Refreshing...
2023-04-18 00:24:09 DEBUG Process-1[checkin_scheduler]: Refreshing headers for current session
2023-04-18 00:24:09 DEBUG Process-1[webdriver]: Starting webdriver for current session
2023-04-18 00:24:12 DEBUG Process-1[webdriver]: Loading Southwest Check-In page
2023-04-18 00:24:24 DEBUG Process-1[webdriver]: Filling out a check-in form to get valid headers
2023-04-18 00:24:35 DEBUG Process-1[webdriver]: Setting valid headers from previous request
2023-04-18 00:24:35 DEBUG Process-1[checkin_scheduler]: Retrieving reservation info from confirmation number
2023-04-18 00:24:39 DEBUG Process-1[checkin_scheduler]: Successfully retrieved reservation info
2023-04-18 00:24:39 DEBUG Process-1[checkin_scheduler]: 2 flights found under current reservation
2023-04-18 00:24:41 DEBUG Process-1[checkin_scheduler]: New flight found. Handling check-in
2023-04-18 00:24:41 DEBUG Process-1[checkin_handler]: Scheduling check-in for current flight
2023-04-18 00:24:41 DEBUG Process-1[notification_handler]: Sending new flights notification
2023-04-18 00:24:42 DEBUG Process-1:1[checkin_handler]: Sleeping until ten minutes before check-in...
2023-04-18 20:09:55 DEBUG Process-1:1[checkin_scheduler]: Refreshing headers for current session
2023-04-18 20:09:55 DEBUG Process-1:1[webdriver]: Starting webdriver for current session
2023-04-18 20:09:58 DEBUG Process-1:1[webdriver]: Loading Southwest Check-In page
2023-04-18 20:10:09 DEBUG Process-1:1[webdriver]: Filling out a check-in form to get valid headers
2023-04-18 20:10:20 DEBUG Process-1:1[webdriver]: Setting valid headers from previous request
2023-04-18 20:10:21 DEBUG Process-1:1[checkin_handler]: Sleeping until check-in: 573 seconds...
2023-04-18 20:19:55 DEBUG Process-1:1[checkin_handler]: Attempting to check in
2023-04-18 20:19:55 DEBUG Process-1:1[checkin_handler]: Making GET request to check in
2023-04-18 20:20:05 DEBUG Process-1:1[checkin_handler]: Making POST request to check in
2023-04-18 20:20:08 DEBUG Process-1:1[checkin_handler]: Successfully checked in!
2023-04-18 20:20:09 DEBUG Process-1:1[notification_handler]: Sending successful check-in notification...

**Comment by jdholtz** [comment]
This is a duplicate of #229. Can you try installing the requirements with `pip3 install -r requirements.txt`? You may have installed the dependencies for Python 2.7 (I also recommend creating a virtual environment to do this). If that doesn't fix it, could you send the output of `pip3 freeze` here?

**Comment by TeSoad90** [comment]
Seriously thank you for your code, and how you make yourself available to troubleshoot. I followed your second option and installed hyper-v to run the code in a virtual environment. The code worked perfectly! I assume like you said I had 2.7 dependencies on my host OS.
